source/training/kickoff.md

```diff
@@ -0,0 +1,123 @@
+# Kick-off
+
+<div class="present"><div class="reveal"><div class="slides">
+
+  <section class="has-dark-background" data-background="#240b35" data-background-image="/_static/img/openfisca-bg.svg" data-background-position="230% 50%" data-background-size="auto 120%">
+    <h2>Kick off</h2>
+    <p>
+      <small>(Rules as Code + OpenFisca) training series</small>
+    </p>
+    <p><a href="https://openfisca.org" class="logo"><img src="/_static/img/openfisca.svg" alt="OpenFisca Logo" ></a></p>
+    <p><small class="author">created by <a href="https://hamish.dev">Hamish Fraser</a></small></p>
```

Comment:
Didn't I deliver the kickoff? 🙂

Reply:
My dear sir, you certainly did, I am no longer so frivolously copy and pasting.